gm4_block_compressors/data/gm4_block_compressors/functions/item/compress.mcfunction

```diff
@@ -0,0 +1,23 @@
+#@s = item entity to be compressed
+#run from block_compressors:item
```

Comment:
Comments should be formatted as `# <comment>`.
Heads should also include the execution position `# at <location>` as well as - preferably - a short description of the function.